data/data-pipeline/data_pipeline/tests/sources/national_risk_index/test_etl.py

```diff
@@ -1,17 +1,58 @@
+from unittest import mock
+
+import filecmp
 import pandas as pd
+import requests
 
 from data_pipeline.config import settings
+from data_pipeline.etl.base import ValidGeoLevel
 from data_pipeline.tests.conftest import copy_data_files
 from data_pipeline.etl.sources.national_risk_index.etl import (
     NationalRiskIndexETL,
 )
+from data_pipeline.tests.sources.test_etl_base import TestETL
+from data_pipeline.utils import get_module_logger
 
-DATA_DIR = (
-    settings.APP_ROOT / "tests" / "sources" / "national_risk_index" / "data"
-)
+logger = get_module_logger(__name__)
+
+
+class TestNationalRiskIndexETL(TestETL):
+    _ETL_CLASS = NationalRiskIndexETL
+    _DATA_DIRECTORY_FOR_TEST = (
+        settings.APP_ROOT / "tests" / "sources" / "national_risk_index" / "data"
+    )
+
+    def _setup_etl_instance_and_run_extract(
+        self, mock_etl, mock_paths
+    ):
+        with mock.patch('data_pipeline.utils.requests') as requests_mock:
```

Comment:
moved this from being a decorator to being a context manager so that the signature of `_setup_etl_instance_and_run_extract` is identical between the base and child classes. 

Also sadly @widal001 I don't think this makes sense to move to conftest.py, because we're monkeypatching `requests` in a very specific way that's going to return the NRI data, so I don't think it's useful on other classes. Do you agree?